packs/data/feats.db/munitions-crafter.json

```diff
@@ -31,15 +31,22 @@
                 "isDailyPrep": true,
                 "key": "CraftingEntry",
                 "maxItemLevel": 1,
-                "requiredTraits": [
-                    [
-                        "alchemical",
-                        "bomb"
-                    ],
-                    [
-                        "alchemical"
+                "formulaPredicate": {
+                    "any": [
+                        {
+                            "and": [
+                                "alchemical",
+                                "bomb"
+                            ]
+                        },
+                        {
+                            "and": [
+                                "alchemical",
+                                "ammunition"
```

Comment:
This will probably need migrating again in https://github.com/foundryvtt/pf2e/pull/3353